src/plugins/postgres_table_copy_operator.py

```diff
@@ -119,40 +120,44 @@ def __init__(
     def _drop_tables_if_unequal(self, cursor: Any, all_table_copies: List[TableMapping]) -> None:
         """Drop the target tables that are unequal to their source table."""
         for table_mapping in all_table_copies:
+            # First make sure the target table exists
+            cursor.execute(
+                sql.SQL("SELECT to_regclass('{target_table_name}')").format(
+                    target_table_name=sql.Identifier(table_mapping.target)
+                )
+            )
```

Comment:
Are you sure this is necessary? When I run the outer join query with a non-existing table name, I get a false result (assuming the other table exists).

Reply:
The problem is, false result is interpreted as tables are not equal. So next step is to remove the target table. However, that does not exist, leading to crash in postgres.

Reply:
Oh, right. Ok!